svg_wheel.py

```diff
@@ -81,6 +81,10 @@ def add_fraction(wheel, packages, total):
     )
     packages_with_wheels.text = '{0}'.format(wheel_packages)
 
+    title = et.SubElement(packages_with_wheels, 'title')
+    percentage = '{:.0%}'.format(wheel_packages/float(total))
```

Comment:
I don't think the explicit cast to `float` is necessary.

Reply:
Yeah, it's only needed for Python 2:
```python
Python 2.7.14 (default, Mar  8 2018, 14:21:32)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> wheel_packages = 20
>>> wheel_packages = 177
>>> total = 360
>>> '{:.0%}'.format(wheel_packages/float(total))
'49%'
>>> '{:.0%}'.format(wheel_packages/total)
'0%'
>>>
```

Python 3 doesn't need it:
```python
Python 3.6.4 (default, Mar  1 2018, 18:36:50)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> wheel_packages = 177
>>> total = 360
>>> '{:.0%}'.format(wheel_packages/float(total))
'49%'
>>> '{:.0%}'.format(wheel_packages/total)
'49%'
```

he `float` can go if you're only using Python 3, and I see you only have 3.6 in .travis.yml, so I'll remove it.